ast/parser.go

```diff
@@ -699,6 +699,11 @@ func (p *Parser) parseElse(head *Head) *Rule {
 	rule.SetLoc(p.s.Loc())
 
 	rule.Head = head.Copy()
+	for i := range rule.Head.Args {
+		if v, ok := rule.Head.Args[i].Value.(Var); ok && v.IsWildcard() {
+			rule.Head.Args[i].Value = Var(p.genwildcard())
+		}
+	}
```

Comment:
The copy here had been misleading the "wildcard unmangling" logic in the formatter: https://github.com/open-policy-agent/opa/blob/main/format/format.go#L168

It picked up the `else`'s invisible args and counted them as a second use that would necessitate the unmangling. It really doesn't, though, and is best sidestepped by introducing fresh wildcard variables for the else args in the first place.